.circleci/config.yml

```diff
@@ -20,14 +20,14 @@ jobs:
     environment:
       GOTHEMIS_IMPORT: github.com/cossacklabs/themis/gothemis
       FILEPATH_ERROR_FLAG: /tmp/test_fail
-      VERSIONS: 1.7 1.7.6 1.8 1.8.7 1.9.4 1.10
+      VERSIONS: 1.8 1.8.7 1.9.4 1.10 1.10.3
```

Comment:
It would be better to lead testing versions to a system. 
Do we want to test only latest subversions of each version? If so than `VERSIONS: 1.8.7 1.9.7 1.10.3`.
If we also want to test first release of each version, than `VERSIONS: 1.8 1.8.7 1.9 1.9.4 1.10 1.10.3`.

Reply:
makes sense for me to test only latest (first option). @Lagovas what do u think?

@shadinua will you do a separate PR to master with proposed change?